src/kool-server/kool-server.go

```diff
@@ -16,19 +18,18 @@ var (
 )
 
 type Result struct {
-	AgentId      int64  `json:"agent_id"`
-	ResponseTime int64  `json:"response_time"`
-	Url          string `json:"url"`
+	AgentId		 int64	`json:"agent_id"`
+	ResponseTime int64	`json:"response_time"`
+	Url			 string `json:"url"`
 }
 
 func connectToDb() error {
 	var err error
-	DB, err = sql.Open("postgres", "host=127.0.0.1 port=24810 dbname=monkey user=kool_writer sslmode=disable")
+	DB, err = sql.Open("postgres", "host=127.0.0.1 port=20010 dbname=monkey user=kool_writer sslmode=disable")
```

Comment:
In a following commit, can you replace this with an environment variable or something similar?